src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -278,7 +278,11 @@ class WorkspaceMembersPage extends React.Component {
                                 text: Str.removeSMSDomain(item.displayName),
                                 alternateText: Str.removeSMSDomain(item.login),
                                 participantsList: [item],
-                                icons: [ReportUtils.getAvatar(item.avatar, item.login)],
+                                icons: [{
+                                    source: ReportUtils.getAvatar(item.avatar, item.login),
+                                    name: item.login,
```

Comment:
Coming from https://github.com/Expensify/App/issues/29645
Not sure if this issue was present at the time this PR was created, but we should have used `formatPhoneNumber` from `withLocalize`
```js
name: props.formatPhoneNumber(details.login),
```